src/SFA.DAS.EAS.Portal/Application/Commands/AddReservationCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.CosmosDb;
+using SFA.DAS.EAS.Portal.Database;
+using SFA.DAS.EAS.Portal.Database.Models;
+using SFA.DAS.Reservations.Messages;
+
+namespace SFA.DAS.EAS.Portal.Application.Commands
+{
+    public class AddReservationCommand
+    {
+        private readonly IAccountsRepository _accountsRepository;
+        private readonly ILogger<AddReservationCommand> _logger;
+
+        public AddReservationCommand(IAccountsRepository accountsRepository, ILogger<AddReservationCommand> logger)
+        {
+            _accountsRepository = accountsRepository;
+            _logger = logger;
+        }
+
+        public async Task Execute(ReservationCreatedEvent reservedFunding, string messageId, CancellationToken cancellationToken = default)
+        {
+            _logger.LogInformation("Executing AddReservationCommand");
```

Comment:
No need to log this.  This type of logging could be added as a decorator of this method (if it was from an interface).  That would prevent mixing of concerns and make unit testing easier. Add as tech debt in 454.